src/__tests__/snackbar-test.tsx

```diff
@@ -235,6 +235,33 @@ test('useSnackbar: happy case', async () => {
     expect(await screen.findByText('any-message')).toBeInTheDocument();
 });
 
+test('useSnackbar: no act warnings when timers are run', async () => {
+    const ComponentWithSnackbar = () => {
+        const {openSnackbar} = useSnackbar();
+        return (
+            <button
+                onClick={() => {
+                    openSnackbar({message: 'any-message'});
+                }}
+            >
+                Open Snackbar
+            </button>
+        );
+    };
+
+    render(
+        <ThemeContextProvider theme={makeTheme()}>
+            <ComponentWithSnackbar />
+        </ThemeContextProvider>
+    );
+
+    await userEvent.click(screen.getByRole('button', {name: 'Open Snackbar'}));
+
+    expect(await screen.findByText('any-message')).toBeInTheDocument();
+
+    await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
why? after a second the snackbar is still open. Perhaps you should manually close it

Reply:
This waiting time let's the 50ms `setTimeout` in the snackbar trigger. If you remove https://github.com/Telefonica/mistica-web/pull/1031#discussion_r1498888232 you will see the test fails.

Reply:
This is the setTimeout I'm talking abount:
```
const openTimeout = setTimeout(() => {
                setIsOpen(true);
            }, 50);
```

Reply:
The timeout time when NODE_ENV==='test' is 0, and when the component is transitioning, it is set as aria-hidden, so, when you find an element inside the snackbar, it is because the snackbar has finished its transition and it is ready to be used.

See the other tests in this suite:
`await userEvent.click(await screen.findByRole('button', {name: 'action'}));`

Reply:
Yeah, there you are querying for a button, but my test case does not have buttons, it's just a snackbar with some text :'(

Reply:
I mean: From what I know `aria-hidden` attribute only hides elements from `byRole` queries @pladaria